kolibri/plugins/setup_wizard/assets/src/actions.js

```diff
@@ -0,0 +1,24 @@
+const Kolibri = require('kolibri');
+
+const DeviceOwnerResource = Kolibri.resources.DeviceOwnerResource;
+const FacilityResource = Kolibri.resources.FacilityResource;
+
+function createDeviceOwnerAndFacility(store, deviceownerpayload, facilitypayload) {
+  const DeviceOwnerModel = DeviceOwnerResource.createModel(deviceownerpayload);
+  const deviceOwnerPromise = DeviceOwnerModel.save(deviceownerpayload);
+  const FacilityModel = FacilityResource.createModel(facilitypayload);
+  const facilityPromise = FacilityModel.save(facilitypayload);
+  const promises = [deviceOwnerPromise, facilityPromise];
+  Promise.all(promises).then(responses => {
+    // redirect to learn page after successfully created the DeviceOwner and Facility.
+    window.location = window.location.origin.concat('/learn/');
```

Comment:
@indirectlylit Is there a more idiomatic way to do redirects within Vue.js? In this case, I guess it's a page reload anyway, since it's cross-app, but wanted to check.

@66eli77 In any case, it should use URL reversal (`Kolibri.urls["kolibri:learnplugin:learn"]` or something like that).